### ISSUE TITLE
Html border style does not change color when switching to night mode

### ISSUE THREAD

**Issue by takanuva15** [ankidroid/Anki-Android#4577]
###### Research
*Enter an [ x ] character to confirm the points below:*

[x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[x] I have searched for similar existing issues here and on the user forum

###### Reproduction Steps

1. One of my decks has been styled to include scrollable boxes using HTML/CSS. The styling code looks like this:
```
<div style="height:80px; line-height:1.75em; overflow:scroll; 
		padding:5px; border:4px double #000000; text-align:right;
		font-size:20px; direction: rtl">
</div>
```

2. When I use the cards in day mode, the boxes appear black like they should.
3. When I switch to night mode, the borders stay black; thus they are no longer visible. (They should change to a white color along with the text.)

###### Expected Result
In day mode, boxes show up as as they should:
![border issue fixed](https://cloud.githubusercontent.com/assets/6986426/22862567/73ee53ee-f0ff-11e6-873d-26f3c032b3bc.png)

###### Actual Result
In night mode, the borders are no longer visible.
![border issue broken](https://cloud.githubusercontent.com/assets/6986426/22862571/80496fd4-f0ff-11e6-9aa3-d88d306258b7.png)

**Comment by timrae** [comment]
The default color inversion algorithm is only designed to work for simple text on a white background. For anything more complex than that, you'll need to customize the night-mode colors on your template using CSS:

https://github.com/ankidroid/Anki-Android/wiki/Advanced-formatting#customize-night-mode-colors

**Comment by takanuva15** [comment]
Awesome thanks! However, I'm not adept at using CSS (I mostly copy-pasted the above styling code). I tried to do this quick fix within the desktop Anki:
![border issue2 desktop code](https://cloud.githubusercontent.com/assets/6986426/22862815/c2fe7884-f103-11e6-9715-1023859e20a8.PNG)

And I got this result:
![border issue2 card view](https://cloud.githubusercontent.com/assets/6986426/22862838/5c2a59ce-f104-11e6-9803-a1f79e95fc49.png)

How would I put the white border around the text only and not the whole card?
